### PR TITLE
CLI | Increase page limit to 10,000 in results service requests (AST-77955)

### PR DESCRIPTION
**Pull request by AlvoBen**
By submitting a PR to this repository, you agree to the terms within the [Checkmarx Code of Conduct](../docs/code_of_conduct.md). Please see the [contributing guidelines](../docs/contributing.md) for how to create and submit a high-quality PR for this repo.

### Description

> Increase page limit to 10,000 in results service requests

### References

> https://checkmarx.atlassian.net/browse/AST-77955

### Testing

> X

### Checklist

- [x] I have added documentation for new/changed functionality in this PR (if applicable).
- [x] I have updated the CLI help for new/changed functionality in this PR (if applicable).
- [x] All active GitHub checks for tests, formatting, and security are passing
- [x] The correct base branch is being used

### CODE REVIEW

File: internal/wrappers/results-http.go [hunk 1]
```diff
@@ -17,8 +17,8 @@ const (
 	sort                    = "sort"
 	sortResultsDefault      = "-severity"
 	offset                  = "offset"
-	astAPIPageLen           = 1000
-	astAPIPagingValue       = "1000"
+	astAPIPageLen           = 10_000
```
Comment (miryamfoiferCX) [inline internal/wrappers/results-http.go:20]: what is the meaning of this value?10_000

Comment (AlvoBen) [reply internal/wrappers/results-http.go:20]: syntactic sugar for 10000

Comment (AlvoBen) [reply internal/wrappers/results-http.go:20]: for better readbility

Comment (miryamfoiferCX) [reply internal/wrappers/results-http.go:20]: great
so add _ also in the next line astAPIPagingValue = "10000"

**Review by miryamfoiferCX — APPROVED** [review]

**Pull request merged**